llvm/lib/Target/RISCV/RISCVTargetMachine.cpp

```diff
@@ -552,6 +553,7 @@ void RISCVPassConfig::addPreEmitPass() {
       EnableRISCVCopyPropagation)
     addPass(createMachineCopyPropagationPass(true));
   addPass(&BranchRelaxationPassID);
+  addPass(createRISCVLatePeepholeOptPass());
```

Comment:
I'm not sure we should be messing with branches after the branch relaxation pass